Leader.js

```diff
@@ -0,0 +1,20 @@
+const SIZE_OF_ARRAY = +process.argv[2];
+const STRING_ARRAY = process.argv[3].trim().split(',')
+const NUMBER_ARRAY = STRING_ARRAY.map((ele) => { return +ele })
+const TEMP_ARRAY = [...NUMBER_ARRAY]; 
+
+let leaderList = [];
+
+function isLeader(number, arr) {
+   let greaterThanEle = arr.filter((numInArr) => {
+      return numInArr > number
+   })
+   if (greaterThanEle.length > 0) return false
+   else return true
```

Comment:
can't we directly return the condition instead of checking it and then return

Reply:
Yes, will update it.